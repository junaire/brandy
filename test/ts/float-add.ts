console.log(40.2 + 1.8);
