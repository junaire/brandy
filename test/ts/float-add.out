@main {
  v0: float = const 40.2;
  v1: float = const 1.8;
  v2: float = fadd v0 v1;
  print v2;
  v3: int = const 0;
}
